server/models/Image.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const imageSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.SchemaTypes.ObjectId,
+    ref: "user",
+  },
+  cloudinaryId: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  url: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  metadata: {
+    type: Object,
+    default: {},
+  },
+  uploadDate: {
+    type: Date,
+    default: Date.now,
+  },
```

Comment:
Look into mongoose timestamps for this ([docs](https://mongoosejs.com/docs/guide.html#timestamps)).